base_layer/core/src/transactions/transaction_components/transaction_output.rs

```diff
@@ -323,9 +323,6 @@ impl TransactionOutput {
             .as_ref()
             .and_then(|f| f.validator_node_registration())
         {
-            // TODO(SECURITY): Signing this with a blank msg allows the signature to be replayed. Using the commitment
-            //                 is ideal as uniqueness is enforced. However, because the VN and wallet have different
-            //                 keys this becomes difficult. Fix this once we have decided on a solution.
```

Comment:
😬

Reply:
haha yeah, they are now marked as an issue on the dan project to figure out what exactly they want to add there